src/test/resources/features/edit_problem_step.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Edit and Delete Steps
```

Comment:
A feature should not map to scenarios 1:1. Features should be broader. We can have one feature - problem solving,  and scenarios - edit and delete problem solution steps, solve problem in a straightforward way.